backend/src/middlewares/check-params-is-valid/check-params-is-valid.middleware.ts

```diff
@@ -0,0 +1,26 @@
+import { RequestHandler } from 'express';
+import { HttpError } from '~/exceptions/exceptions';
+import { ErrorMessage, HttpCode } from '~/common/enums/enums';
+import { MAX_POSSIBLE_ID } from '~/common/constants/constants';
+
+const checkParamsIsValid = (): RequestHandler => {
```

Comment:
but we can have not only id in parameters
we should create something more specific for it